gradle/dependencies.gradle

```diff
@@ -32,7 +36,9 @@ def build = [
     targetSdkVersion: 26,
 
     checkerFramework: 'org.checkerframework:dataflow:2.3.0',
+    errorProneCheckApi: "com.google.errorprone:error_prone_check_api:${versions.errorProne}",
     errorProne: "com.google.errorprone:error_prone_core:${versions.errorProne}",
```

Comment:
nit: this goes above errorProneCheckApi alphabetically

Reply:
https://github.com/uber/AutoDispose/pull/156/commits/05a3a34b3778da0031e5dfa0161256f224ed6cdf